build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="test/ProjectsToTest.props" />
+  <Import Project="test/TestPackageProjects.props" />
+
+  <Target Name="PrepareTests"
+          DependsOnTargets="Prepare">
```

Comment:
Agreed here. @eerhardt you had previously mentioned init was becoming a grab bag. I wonder if it will make sense to break it up as we move forward as well as a rename.